src/scripts/clipperUI/componentBase.ts

```diff
@@ -67,7 +67,7 @@ export abstract class ComponentBase<TState, TProps> {
 	 * Example use:
 	 *      <a id="myCoolButton" {...this.enableInvoke(this.myButtonHandler, 0)}>Click Me</a>
 	 */
-	public enableInvoke(handleMethod: Function, tabIndex = 0, args?: any, idOverride: string = undefined) {
+	public enableInvoke(handleMethod: Function, tabIndex = 0, args?: any, idOverride: string = undefined, valueOfSetNameForArrowKeyNav: string = undefined) {
```

Comment:
Note that enableInvoke returns a list of additional attributes to add to the element it is called on e.g., onclick, tabIndex. We can use the value of your new param to add the setNameForArrowKeyNav attribute to the element automatically, so elements don't have to add it manually as well (which is duplicate work)